vector/v.info/local_proto.h

```diff
@@ -5,6 +5,8 @@
 #define SHELL_BASIC  0x02
 #define SHELL_REGION 0x04
 #define SHELL_TOPO   0x08
+#define STR_LEN      1001
+#define MAX_STR_LEN  3003
```

Comment:
You can use [GPATH_MAX](https://grass.osgeo.org/programming8/gis_8h.html#a8528103327f48c05298443f89a7dd353) instead (no need for defining new macro).